### ISSUE TITLE
@Value and @Wither not playing well together

### ISSUE THREAD

**Issue by rcaval** [mplushnikov/lombok-intellij-plugin#96]
The issue I'm facing is that `@Wither` seems to require `@AllArgsConstructor` to be recognized by the IDE. If I only have `@Value`, which already creates a meaningful contructor it doesn't work properly.

The bellow code compiles fine, but IntelliJ keeps complaining about it. Same occurs if I add `@RequiredArgsConstructor(access = AccessLevel.PRIVATE)` in order to restrict my constructor visibility.

I'm running lombok plugin v0.9.2. I think it was introduces in that version, because I haven't noticed that before.

```
@Wither
@Value
public class Dummy {

    private final String myField;

    public void methodCallingWith(){
        this.withMyField("");
    }
}
```

**Comment by mplushnikov** [comment]
Should be fixed in the next release 0.9.3

**Comment by rcaval** [comment]
Great. Just got it.
